packages/unmock-core/src/service/dsl/actors.ts

```diff
@@ -1,25 +1,29 @@
 import debug from "debug";
```

Comment:
If you're looking for a good FP start, this is a great function to do it in - it can be rewritten to create a new object with only the `.default` parameter changed and will probably be more performant than the deep copy (although we are talking about single-digit microseconds in the grand scheme of things, so the only real reason to do it would be to have fun with FP).

Reply:
Could you elaborate? The`default` is not needed in the copied version (which is later used to generate the response). However, the DSL does not interact with a `State` object immediately, so it modifies a passing state as needed (reduces the `default` value for the `SCHEMA_TIMES` property).

Could remove the `delete` statement if I could copy anything *but* that specific key. It seems easier to do a deep copy; do you have any suggestions?

Reply:
`const {x, ...y} = {x: 5, a: 2, b: 3 }` works only with hard coded names, otherwise that would've been ideal :thinking:

Reply:
Ah, managed to achieve this with some destructuring (removing the deep copy and the last `delete` statement).

There is still a `delete` statement to modify the `State`'s state as needed. I think that's fine for now (at least the topic for a different PR, as it would require bigger changes).

Reply:
Not sure if it helps but lodash has `_.omit` and `_.omitBy` that could help avoid the deletes.